test/src/unit-cppapi-array.cc

```diff
@@ -1326,46 +1362,54 @@ TEST_CASE(
   Query query_r(ctx, array_r, TILEDB_READ);
 
   SECTION("Non empty range") {
-    query_r.add_range(0, s1, s2);
-    CHECK_THROWS(query_r.add_range(1, s1, s2));
+    Subarray subarray_r(ctx, array_r);
```

Comment:
Nit: instantiate `Subarray subarray_r(ctx, array_r);` on L1363 and remove from each section